docs/Database_Schema.md

```diff
@@ -575,30 +588,139 @@ CREATE TABLE IF NOT EXISTS "fees"(
+| proposal_result     | TEXT      |     |          |         | Proposal result               |
+
+### _Create Script_
+
+```sql
+CREATE TABLE IF NOT EXISTS "proposals"(
+    "proposal_id"          TEXT        NOT NULL,
+    "proposer_address"     TEXT        NOT NULL,
+    "tx_hash"              TINYBLOB    NOT NULL,
+    "title"                TEXT        NOT NULL,
+    "type"                 TEXT        NOT NULL,
+    "status"               TEXT        NOT NULL,
+    "voting_start"         DATETIME    NOT NULL,
+    "voting_end"           DATETIME    NOT NULL,
+    "submit_time"          TEXT        NOT NULL,
+    "detail"               TEXT        NOT NULL,
+    "fee_tx_hash"          TEXT        NOT NULL,
+    "vote_fee"             BigInt(20)  NOT NULL,
+    "funding_amount"       BigInt(20)  NOT NULL,
+    "voting_start_height"  INTEGER     NOT NULL,
+    "voting_end_height"    INTEGER     NOT NULL,
+    "proposer_id"          INTEGER     NOT NULL,
+    "proposal_result"      TEXT,
+
+    PRIMARY KEY ("proposal_id(64)");
+);
+```
+---
+
+## 23. Table **proposer**
+
+### _Schema_
+
+| Column              | Data Type | PK  | Not NULL | Default | Description                   |
+| :------------------ | :-------- | :-: | :------: | ------- | ----------------------------- |
+| proposer_id         | TEXT      |  Y  |    Y     |         | Proposer id                   |
+| proposer_name       | TEXT      |     |    Y     |         | Proposer name                 |
+| wallet_address      | TEXT      |     |    Y     |         | Wallet address                |
+
+### _Create Script_
+
+```sql
+CREATE TABLE IF NOT EXISTS "proposer"(
+    "proposer_id"       TEXT    NOT NULL,
+    "proposer_name"     TEXT    NOT NULL,
+    "wallet_address"    TEXT    NOT NULL,
+    PRIMARY KEY ("proposer_id(64)");
+);
+```
+---
+
+## 24. Table **vote**
+
+### _Schema_
+
+| Column              | Data Type | PK  | Not NULL | Default | Description                   |
+| :------------------ | :-------- | :-: | :------: | ------- | ----------------------------- |
+| vote_id             | INTEGER   |  Y  |    Y     |         | Vote id                       |
+| proposal_id         | TEXT      |  Y  |    Y     |         | Proposal id                   |
+| app_name            | TEXT      |     |    Y     |         | App name                      |
+| voter_utxo          | TINYBLOB  |     |    Y     |         | Voter UTXO key                |
+| tx_hash             | TINYBLOB  |     |    Y     |         | Vote tx hash                  |
+| voter_address       | TEXT      |     |    Y     |         | Voter address                 |
+| sequence            | INTEGER   |     |    Y     |         | Sequence                      | 
+| ballot_answer       | BLOB      |     |    Y     |         | Ballot answer                 |
+| voting_time         | TEXT      |     |    Y     |         | Voting time                   |
```

Comment:
We don't know the exact voting_time, but we know the block height that contains the transaction.
Each block has a block timestamp.

Reply:
resloved